smartclide-monitoring/src/main/java/de/atb/context/monitoring/analyser/GitlabCommitAnalyser.java

```diff
@@ -46,20 +47,20 @@ public GitlabCommitAnalyser(final DataSource dataSource,
         if (!(dataSource instanceof GitlabDataSource)) {
             throw new IllegalArgumentException("Given dataSource must be of type GitlabDataSource!");
         }
-        gitRestCallService = new GitRestCallService(((GitlabDataSource) dataSource).getGitLabAccessToken());
+        gitlabApiClient = new GitlabApiClient(((GitlabDataSource) dataSource).getGitLabAccessToken(), gitlabBaseUri);
```

Comment:
You don't want to have the gitlab URI hard-coded. What if we want to monitor <https://gitlab.com/> in the future?

The `DataSource` has a `uri` property, which allows to configure the URI to be monitored through the `monitoring-config.xml` configuration file.

You'd want to use `dataSource.getUri()` here.